data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```diff
@@ -78,50 +149,56 @@
           "path": "observedTimestamp"
         },
         "traceId": {
-          "ignore_above": 256,
-          "type": "keyword"
+          "type": "keyword",
+          "ignore_above": 32
         },
         "spanId": {
-          "ignore_above": 256,
-          "type": "keyword"
+          "type": "keyword",
+          "ignore_above": 16
         },
         "flags": {
-          "type": "integer"
+          "type": "long"
         },
         "schemaUrl": {
-          "type": "keyword"
+          "type": "keyword",
+          "ignore_above": 256
         },
-        "instrumentationScope": {
-          "properties": {
-            "name": {
-              "type": "keyword"
-            },
-            "version": {
-              "type": "keyword"
-            }
-          }
+        "serviceName": {
+          "type": "alias",
+          "path": "resource.attributes.service.name"
         },
         "event": {
```

Comment:
Why do we need event here?
https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto
We don't have anything in logs proto model.

Reply:
That is a question I am wondering myself. This comes out of the [Simple Schema for Observability](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/schema/observability/logs/README.md#event-fields). I kept it, since it was in the original file proposed by @kkondaka. I know this is not in the OpenTelemetry standard. The same is true for the `data_stream`. How do we want to go about this?